ractive.js

```diff
@@ -0,0 +1,30 @@
+var people = [{
+    name: 'Subash',
+    age: 29,
+    isMinor: false
+  },
+  {
+    name: 'Sunil',
+    age: 22,
+    isMinor: false
+  },
+  {
+    name: 'Nita',
+    age: 19,
+    isMinor: true
+  }
+];
+
+var ractive = new Ractive({
```

Comment:
You should not have 2 Ractive instances. However, you have this one and the one in ractive.html.